src/utils/arch/cpu.h

```diff
@@ -44,6 +44,7 @@ typedef enum ucc_cpu_vendor {
     UCC_CPU_VENDOR_AMD,
     UCC_CPU_VENDOR_GENERIC_ARM,
     UCC_CPU_VENDOR_GENERIC_PPC,
+    UCC_CPU_VENDOR_GENERIC_RISCV,
```

Comment:
please update copyright header

Reply:
Thanks, done. Also adjusted the commit title to make the CI happy.